server/Labels/messageBus.js

```diff
@@ -0,0 +1,25 @@
+var messageBus = {
```

Comment:
This is actually my mistake, I don't know how this file keeps getting into my repo, this was a toy problem I accidently saved here, I'll get rid of it